collections/_article/dev-snapshot-godot-4-2-dev-5.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 5"
+excerpt: "Penultimate 4.2 dev snapshot brings improvements to ..."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-5.webp
+image_caption_title: "Starter Kit 3D Platformer"
+image_caption_description: "A template project by Kenney"
+date: 2023-09-18 16:00:00
+---
+
+We are entering the final stretch of the feature development for this dev cycle. There is probably one more Godot 4.2 dev snapshot left before the project would enter the feature freeze and all efforts would turn towards fixing remaining bugs and stabilizing the release. In the [past 2 months](/article/dev-snapshot-godot-4-2-dev-1) contributors have been shaping what Godot 4.2 is going to be, and we will soon pass the mark of [1000 changes](https://godotengine.github.io/godot-interactive-changelog/#4.2).
+
+A significant part of this snapshot contains changes aimed at improving user experience throughout the editor. We are also introducing a new version of Godot Android plugins. As always, many bugs and crashes have been squashed and other issues resolved. Here's a short highlights section, ahead of a more complete list [below](#whats-new):
```

Comment:
```suggestion
A significant part of this snapshot contains changes aimed at improving the user experience throughout the editor. We are also introducing a new version of the Godot Android plugins. As always, many bugs and crashes have been squashed and other issues resolved. Here's a short highlights section, ahead of a more complete list [below](#whats-new):
```

Reply:
It's not about specific plugins, it's about a feature that can be described as Godot Android plugins. So your first suggestion should probably be fine.